### PR TITLE
Update page-report-es.yml

### PR DESCRIPTION
**Pull request by JuanVqz**
### Description

Updating content to the Spanish language.

### Motivation

Make page reports easy to read for Spanish tongue contributors

### Additional details

Thank you @IvanAquino for contributing to this translation.

### Related issues and pull requests

Related to https://github.com/mdn/translated-content/pull/9073, #8918

### CODE REVIEW

**Comment by JuanVqz** [comment]
@mdn/yari-content-es it would be nice if you have a look, and share your thoughts.

File: .github/ISSUE_TEMPLATE/page-report-es.yml [hunk 1]
```diff
@@ -5,58 +5,58 @@ body:
   - type: markdown
     attributes:
       value: |
-        ### Before you start
+        ### Antes de comenzar
 
-        **Want to change this page yourself?** This content is open source!
-        ↩ Go back and use the _Edit on GitHub_ link on the page.
+        **¿Quiere cambiar esta página por usted mismo?** ¡Este contenido es de código abierto!
+        ↩ Regrese y use el enlace _Editar en GitHub_ en la página.
```
Comment (JuanVqz) [inline .github/ISSUE_TEMPLATE/page-report-es.yml:11]:
```suggestion
        ↩ Regrese a la página y use el enlace _Editar en GitHub_.
```

File: .github/ISSUE_TEMPLATE/page-report-es.yml [hunk 2]
```diff
@@ -5,58 +5,58 @@ body:
...
 
-        **Is your issue about the browser compatibility table?**
-        ↩ Go back and use the _Report problems with this compatibility data on GitHub_ link on the page.
+        **¿El problema es acerca de la tabla de compatibilidad con los navegadores?**
+        ↩ Regrese y use el enlace _Reportar problemas con los datos de compatibilidad en GitHub_ en la página.
```
Comment (JuanVqz) [inline .github/ISSUE_TEMPLATE/page-report-es.yml:14]:
```suggestion
        ↩ Regrese a la página y use el enlace _Reportar problemas con los datos de compatibilidad en GitHub_.
```

File: .github/ISSUE_TEMPLATE/page-report-es.yml [hunk 3]
```diff
@@ -5,58 +5,58 @@ body:
...
+        **¿Necesita ayuda con su navegador?**
+        🙋 Para obtener ayuda con [Firefox](https://support.mozilla.org/en-US/kb/file-bug-report-or-feature-request-mozilla), [Chrome](https://support.google.com/chrome/answer/95315?hl=en-GB&ref_topic=7439544), [Safari](https://www.apple.com/feedback/safari.html), o algún otro navegador, visite la página de soporte del navegador.
 
         ---
   - type: input
     id: mdn-url
     attributes:
       label: MDN URL
-      description: Set automatically. Do not modify.
+      description: Se establece automáticamente. No la modifique.
```
Comment (Graywolf9) [inline .github/ISSUE_TEMPLATE/page-report-es.yml:24]: Para usar el infinitivo preferente

```suggestion
      description: Se establece automáticamente. No modificar.
```

File: .github/ISSUE_TEMPLATE/page-report-es.yml [hunk 4]
```diff
@@ -5,58 +5,58 @@ body:
...
-      description: Link to information that helps us confirm your issue.
+      label: ¿Tiene algún enlace de apoyo, referencias o citas?
+      description: Enlace a información que nos ayude a confirmar el problema.
   - type: textarea
     id: more-info
     attributes:
-      label: Do you have anything more you want to share?
-      description: For example, steps to reproduce a bug, screenshots, screen recordings, or sample code
+      label: ¿Tiene algo más que quiera compartir?
+      description: Por ejemplo, pasos para reproducir el error, capturas o grabaciones de pantalla, o algún código de ejemplo.
```
Comment (Graywolf9) [inline .github/ISSUE_TEMPLATE/page-report-es.yml:52]:
```suggestion
      description: Por ejemplo, pasos para reproducir el error, capturas, grabaciones de pantalla, o algún código de ejemplo.
```

File: .github/ISSUE_TEMPLATE/page-report-es.yml [hunk 5]
```diff
@@ -5,58 +5,58 @@ body:
...
       value: |
         ---
-        You're finished! The following fields are prefilled. Please click **Submit new issue**.
+        ¡Haz terminado! Los siguientes campos están prellenados. Por favor haga clic en **Submit new issue**.
   - type: textarea
     id: metadata
     attributes:
       label: MDN metadata
-      description: Set automatically. Do not modify.
+      description: Se establece automáticamente. No la modifique.
```
Comment (Graywolf9) [inline .github/ISSUE_TEMPLATE/page-report-es.yml:62]:
```suggestion
      description: Se establece automáticamente. No modificar.
```

**Comment by Graywolf9** [comment]
Muchas gracias! @JuanVqz te he dejado algunos comentarios

**Review by caugner — COMMENTED** [review]
If @Graywolf9 approves, I can merge.

**Comment by JuanVqz** [comment]
@Graywolf9 cuando tengas tiempo de aprobar el PR por favor?

**Comment by JuanVqz** [comment]
> @Graywolf9 cuando tengas tiempo de aprobar el PR por favor?

@Graywolf9 solo para recordarte de este pr. cuando tengas tiempo de revisarlo/aprobarlo, gracias!

**Review by Graywolf9 — APPROVED** [review]
Thank you so much for all your help! @JuanVqz 

So sorry about the late, when the bot remove the `l10n-es` tag this keeps off of my radar 😅

But this is perfect to merge, thanks for your patience @caugner

**Review by caugner — APPROVED** [review]

**Comment by caugner** [comment]
Great work, @JuanVqz and @Graywolf9! 🎉

**Pull request merged**